### DISCUSSION TITLE
[Bug]: review more than 24hours in review section (at a time)?

### DISCUSSION THREAD

**Discussion by bob454522** [blakeblackshear/frigate#12932] [Report a Bug]
### Describe the problem you are having

Not sure if this is an issue or a new feature, which is why I'm posting it to discussion (not issues)-

  it seems on v14 we can no longer select a range of time to review (on the /review page that is).  (ie if i wanted to select the past 3 days from the time picker, for review)

It seems it's always set to 24 hours and we can only click a prior day and see that 24 hour frame.

is this the new intended behavior?

if not please let me know and i can post a screen cap or more info.  ( did search and could not find anything mentioning this feature change - as i know in prior v14-beta and v13 you could select a timerange or different pre-set times).
thanks

![image](https://github.com/user-attachments/assets/19f9e9a7-d44d-416c-b18f-6136d9d5df25)

### Steps to reproduce

try to select a range of a few days (ie more than 24 hours , at one time) to review on the /review page

### Version

0.14.0-da913d8

### Frigate config file

```yaml
is a GUI question
```

### Relevant log output

```shell
NA
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Axis

**Comment by NickM-27** [comment]
This is working as expected https://github.com/blakeblackshear/frigate/issues/12808
